### PR TITLE
adding disable_active_reparents for vttablet process

### LINKED ISSUE #1094: fix unmanaged tablet docs
https://vitess.io/docs/14.0/user-guides/configuration-advanced/unmanaged-tablet/

We fail to provide `--disable_active_reparents` in the vttablet startup flags in the example in this doc, even though the flag is documented right below the link on this other page https://vitess.io/docs/14.0/reference/programs/vttablet/#external-mysql

Should be fixed in 15.0, 14.0 and as far back as applicable.

Slack conversation:
https://vitess.slack.com/archives/C0PQY0PTK/p1657185901075869

### PR DESCRIPTION
**Pull request by rsajwani**
Fixes #1094 
Signed-off-by: Rameez Sajwani <rameezwazirali@hotmail.com>

### CODE REVIEW

File: content/en/docs/13.0/reference/programs/vttablet.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ See [Unmanaged Tablet](../../../user-guides/configuration-advanced/unmanaged-tab
 
 Even if a MySQL is external, you can still make vttablet perform some management functions. They are as follows:
 
-* `-disable_active_reparents`: If this flag is set, then any reparent or replica commands will not be allowed. These are InitShardMaster, PlannedReparent, PlannedReparent, EmergencyReparent, and ReparentTablet. In this mode, you should use the TabletExternallyReparented command to inform vitess of the current primary.
+* `-disable_active_reparents`: If this flag is set, then any reparent or replica commands will not be allowed. These are InitShardMaster, PlannedReparent, EmergencyReparent, and ReparentTablet. In this mode, you should use the TabletExternallyReparented command to inform vitess of the current primary.
```
Comment (deepthi) [inline content/en/docs/13.0/reference/programs/vttablet.md]:
```suggestion
* `-disable_active_reparents`: If this flag is set, then any reparent or replica commands will not be allowed. These are InitShardPrimary, PlannedReparentShard, EmergencyReparentShard, and ReparentTablet. In this mode, you should use the TabletExternallyReparented command to inform vitess of the current primary.
```

Comment (deepthi) [reply content/en/docs/13.0/reference/programs/vttablet.md]: 1. I'm pretty sure we had already renamed InitShardMaster -> InitShardPrimary by v13, but you should double-check.
2. These changes need to go into all the versions.

Comment (deepthi) [reply content/en/docs/13.0/reference/programs/vttablet.md]: I'm not sure what "replica" commands are meant here. Can you check the blame and see where this line came from?

Comment (rsajwani) [reply content/en/docs/13.0/reference/programs/vttablet.md]: "replica" was introduced [here](https://github.com/vitessio/website/commit/206458d84b1785486a7bd7e29fa3ab79b0cea8fe) ... looks like it was always there.

**Review by deepthi — APPROVED** [review]
Thank you for fixing this, it has been open for a long time.

**Pull request merged**